go-controller/go.mod

```diff
@@ -20,8 +20,8 @@ require (
 	github.com/mitchellh/copystructure v1.2.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.15.0
-	github.com/openshift/api v0.0.0-20220525145417-ee5b62754c68
-	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
+	github.com/openshift/api v0.0.0-09c070622e2c128278ef207780548e2f290d804c
```

Comment:
Great, this is the commit hash of the merge pull request for the 1.25 rebase in openshift/api: https://github.com/openshift/api/commit/09c070622e2c128278ef207780548e2f290d804c